pkg/dataloaders/cedar_system_loader.go

```diff
@@ -0,0 +1,74 @@
+package dataloaders
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/graph-gophers/dataloader"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+const cedar_system_loader_key = "id"
+
+func (loaders *DataLoaders) getCedarSystemBatchFunction(ctx context.Context, keys dataloader.Keys) []*dataloader.Result {
+	logger := appcontext.ZLogger(ctx)
+	cedarSystems, err := loaders.GetCedarSystems(ctx)
+	if err != nil {
+		logger.Error("error getting cedar systems in data loader", zap.Error(err))
```

Comment:
we should return a slice of `dataloader.Result` with every `err` populated with `err` - i believe dataloaders always need to return a slice with `len` of requested `keys`

Reply:
oooooh, ok

Reply:
Added in [1c15a9f](https://github.com/CMSgov/easi-app/pull/2577/commits/1c15a9f4187febe1f908bb6481a313287c1236df)